pkg/registration/hub/manifests/rbac/managedcluster-registration-clusterrole.yaml

```diff
@@ -11,7 +11,7 @@ rules:
   # TODO for backward compatible, we do not limit the resource name
   # remove this after we no longer support lower versions kubernetes (less than 1.14)
   #resourceNames: ["managed-cluster-lease"]
-  verbs: ["get", "update"]
+  verbs: ["get", "update", "list", "watch"]
```

Comment:
why need list/watch here?

Reply:
the hubtimeout controller needs to check the lease every minutes since we mentioned the apiserver call pressure, I think we should using lister for this.

Reply:
Removed, use kubeclient instead of lister in hubtimeout controller.